### PR TITLE
Getty Images in providers

### PR DESCRIPTION
**Pull request by bbashel1**
add Getty Images in provider

### CODE REVIEW

File: src/GettyImages/Provider.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+   }
+
+   /**
+    * {@inheritdoc}
+    */
+   protected function mapUserToObject(array $user)
+   {
+       return (new User)->setRaw($user)->map([
+           'first_name' => $user['first_name'],
+           'last_name'     => $user['last_name'],
```
Comment (atymic) [inline src/GettyImages/Provider.php:89]: alignment here is a bit wonky.

**Comment by bbashel1** [comment]
added docs
https://github.com/SocialiteProviders/Documentation/pull/86

**Comment by bbashel1** [comment]
added to the split config

**Comment by atymic** [comment]
Can you fix the style issues?

**Pull request merged**